### PR TITLE
[EA] Topic relevance vote redesign

### PR DESCRIPTION
**Pull request by oetherington**
This PR adds the new designs for tag relevance voting:
Screenshot 2023-05-15 at 17 46 35

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204513372557541) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by agnesstenlund** [comment]
Alright so I think we could either just bring back the old style in the new place (ignore old bookmarks)
Screenshot 2023-05-08 at 20 04 37

Alternatively we make them look different to the new upvote icon (and instead make it look like the old one, which we still use on the post page)
Screenshot 2023-05-08 at 20 04 18

In this case it'd be nice to pretty soon update the UI of the upvote button on the post page (would be nice to do either way).

@oetherington I'm guessing the second option is (a lot?) more work? If that's the case, I think it'd be fine just bringing back the old style. It's not great that it looks like an upvote button, but since it behaves like one (with the strong votes) it may still make most sense.

Another thing is that it looks slightly different on the core topic pages vs. normal topic pages:
Screenshot 2023-05-08 at 19 57 46
Screenshot 2023-05-08 at 19 57 23

fyi @jpaddison3

**Comment by jpaddison3** [comment]
I think bringing back the old style is a good hotfix.

I _would_ love to see the beautiful new - + style implemented with strong voting at some point, but probably not worth a major design effort right now.

**Comment by oetherington** [comment]
@agnesstenlund Just pushed the changes - let me know if there's anything else!

I ended up using the arrow styles to match the posts page - it's only a one line change to switch between the two styles.

Screenshot 2023-05-15 at 17 46 35

**Comment by agnesstenlund** [comment]
Ohh this looks perfect, thanks @oetherington!

I found one case where it just doesn't show up, do you know why?
Screenshot 2023-05-16 at 11 56 37

File: packages/lesswrong/components/tagging/TagPage.tsx [hunk 1]
```diff
@@ -161,14 +175,46 @@ export const tagPostTerms = (tag: Pick<TagBasicInfo, "_id" | "name"> | null, que
...
+      <>
+        <SectionTitle title={`Posts tagged ${tag.name}`} />
+        <div className={classes.postListMeta}>
+          <PostsListSortDropdown value={query.sortedBy || "relevance"} />
+          <div className={classes.relevance}>Relevance</div>
+        </div>
+      </>
+    );
+  }
+  if (tag.sequence) {
```
Comment (jpaddison3) [inline packages/lesswrong/components/tagging/TagPage.tsx:195]: Intro sequences aren't used on lesswrong, I would cut this conditional.

File: packages/lesswrong/components/tagging/subforums/SubforumWikiTab.tsx [hunk 2]
```diff
@@ -135,18 +126,11 @@ const SubforumWikiTab = ({tag, revision, truncated, setTruncated, classes}: {
...
-                  Posts tagged <em>{tag.name}</em>
-                </div>
-                <PostsListSortDropdown value={query.sortedBy || "relevance"} />
-              </div>
-            )}
+            <SectionTitle title={`Posts tagged ${tag.name}`} />
+            <div className={classes.postListMeta}>
+              <PostsListSortDropdown value={query.sortedBy || "relevance"} />
+              <div className={classes.relevance}>Relevance</div>
+            </div>
```
Comment (jpaddison3) [inline packages/lesswrong/components/tagging/subforums/SubforumWikiTab.tsx:133]: Ew, are we really still copy pasting this file? Sigh.

**Review by jpaddison3 — APPROVED** [review]
ASTLAC wrt my and Agnes's comments. Excited to fix this!

**Comment by oetherington** [comment]
@agnesstenlund Great - thanks!

That post is just broken on the dev database (there's quite a few like this) - in theory that should never happen in production. Hopefully we'll refresh the dev database which fill fix a lot of little annoying problems like this.

File: packages/lesswrong/components/tagging/subforums/SubforumWikiTab.tsx (hunk 2 above)
Comment (oetherington) [reply packages/lesswrong/components/tagging/subforums/SubforumWikiTab.tsx:133]: Yeah, honestly I have no idea what's going on with the structure here - I think it's refelecting a page taxonomy that no longer exists(?) which is kind of confusing.

**Pull request merged**